css/style.css

```diff
@@ -0,0 +1,291 @@
+* {
+   box-sizing: border-box;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)